_component_design/tables-controls.md

```diff
@@ -11,6 +11,7 @@ resource: true
 last-modified: 2017-10-24
 helix-ui-css: false
 helix-ui-javascript: false
+pageLink: none
```

Comment:
(up to you) closest equivalent is the Pagination component (`components/pagination/`)

Reply:
will wait on this.  Pagination needs to be its own design doc, separate from tables